README.md

```diff
@@ -123,15 +123,15 @@ Google Cloud BigQuery (Alpha)
 
 Here is a code snippet showing a simple usage example from within Compute/App Engine. Note that you
 must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+Complete source code can be found at
+[CreateTableAndLoadData.java](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/snippets/CreateTableAndLoadData.java).
 
 ```java
 import com.google.gcloud.bigquery.BigQuery;
```

Comment:
I think there's some value in having imports there, just to make it easier for an eager-to-start user to start playing with some code. But I agree with you that having the complete source code link makes them less needed so I don't mind removing them.

I would keep as is the more "interactive" presentation we have in each module's README, where we outline code and imports for each micro-functionality.